subiquity/ui/views/mirror.py

```diff
@@ -227,17 +228,19 @@ async def confirm_continue_anyway() -> None:
             )
 
             if confirmed:
-                self.controller.done(result.url.value)
+                skip_archive = True if continue_skip_archive else None
```

Comment:
The True/False/None tristate is a bit confusing but well. One thing at a time.

Reply:
Sorry about that :upside_down_face: